### PR TITLE
Fix for pull request #13596 missing a hidden xs attribute

### LINKED ISSUE #13577: [FRIO] Remove the icon descriptions for the interaction options on posts.
### Describe the feature you'd like
The icon descriptions for the interaction options on posts (like, dislike, share, quote) are more of a hindrance than a help in deep discussions. It makes the application unnecessarily complex and confusing. They were originally helpful to ensure accessibility. Tooltips continue to ensure accessibility.

In comparable applications such as Friendica, the description on icon is dispensed with. In the mobile view, the icon description is also dispensed with in Friendica.

grafik
grafik
grafik
grafik

### PR DESCRIPTION
**Pull request by Raroun**
Readded the hidden-xs attribute which caused double buttons on mobile-view.

### CODE REVIEW

File: view/theme/frio/templates/wall_thread.tpl [hunk 1]
```diff
@@ -303,8 +303,10 @@ as the value of $top_child_total (this is done at the end of this file)
 		<!-- <hr /> -->
 		<div class="wall-item-actions">
 			{{* Action buttons to interact with the item (like: like, dislike, share and so on *}}
-
 			{{* Buttons for like and dislike *}}
+			<div class="wall-item-actions-items btn-toolbar btn-group hidden-xs" role="group">
+			<div class="wall-item-actions-row">
```
Comment (MrPetovan) [inline view/theme/frio/templates/wall_thread.tpl]: Indentation
```suggestion
				<div class="wall-item-actions-row">
```

File: view/theme/frio/templates/wall_thread.tpl [hunk 2]
```diff
@@ -349,7 +351,7 @@ as the value of $top_child_total (this is done at the end of this file)
 			{{* Put additional actions in a dropdown menu *}}
 				<span role="presentation" class="separator"></span>
 				<span class="more-links btn-group{{if $item.thread_level > 1}} dropup{{/if}}">
-                                    <button type="button" class="btn-link dropdown-toggle" data-toggle="dropdown" id="dropdownMenuOptions-{{$item.id}}" aria-haspopup="true" aria-expanded="false" title="{{$item.menu}}"><i class="fa fa-ellipsis-h" aria-hidden="true"></i>&nbsp;{{$item.menu}}</button>
+                                    <button type="button" class="btn-link dropdown-toggle" data-toggle="dropdown" id="dropdownMenuOptions-{{$item.id}}" aria-haspopup="true" aria-expanded="false" title="{{$item.menu}}"><i class="fa fa-ellipsis-h" aria-hidden="true"></i></button>
```
Comment (MrPetovan) [inline view/theme/frio/templates/wall_thread.tpl]: Indentation
```suggestion
					<button type="button" class="btn-link dropdown-toggle" data-toggle="dropdown" id="dropdownMenuOptions-{{$item.id}}" aria-haspopup="true" aria-expanded="false" title="{{$item.menu}}"><i class="fa fa-ellipsis-h" aria-hidden="true"></i></button>
```

File: view/theme/frio/templates/wall_thread.tpl [hunk 3]
```diff
@@ -445,9 +447,9 @@ as the value of $top_child_total (this is done at the end of this file)
                                                 <a class="btn-link navicon ignore" href="{{$item.report.href}}"><i class="fa fa-flag" aria-hidden="true"></i> {{$item.report.label}}</a>
                                         </li>
                                         {{/if}}
-                                    </ul>
-                        </span>
-
+                </span>                 </ul>
+			</div>
+			</div>
```
Comment (MrPetovan) [inline view/theme/frio/templates/wall_thread.tpl]: Please respect the original markup, also indentation.
```suggestion
                                    </ul>
				</span>
				</div>
			</div>
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by Raroun** [comment]
thank you, I will watch out for it from now on.

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
For next time, you can add all the suggested changes to the same commit in the "Files" tab. Instead of clicking "Commit the changes" on each suggestion, you can click "Add to commit" and then "Commit the changes" at the top where it should show how many suggestions were selected.

**Pull request merged**